scripts/install.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+# runs setup of base environment and database installs
+
+# create core conda env
+environment_yml = '%s/../%s.yml' % (os.__file__, 'environment')
+os.call('conda env create --name oecophylla -f %s' % environment_yml)
```

Comment:
why not just call the install script with the changes from this morning?

Reply:
this PR won't be performing install functionality - so this is safe to ignore